def do(num):
    try:
        if num:
            return int(num) + 5
        else:
            print('Return a no')
    except ValueError as err:
        return err
    except ValueError as err:
        return err

